starter-code/AboutPage.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Rawan Alrajeh | Portfolio</title>
+        <link rel="stylesheet" href="../starter-code/Css/Style-of-About.css">
+    </head>
+<body>
+    <p>&nbsp;</p>
+<header>
+	
+<div class="isheader" style="text-align: center;"><span style="color: #666699;"><strong><a style="color: #666699;" href="#">About Me</a> | <a style="color: #666699;" href="../starter-code/Portfolio.html">Portfolio</a></strong></span></div>
```

Comment:
avoid using inline styling unless you really find it necessary.

keep all styling in the CSS file.